tests/core/pyspec/eth2spec/test/eip7251/block_processing/test_process_deposit.py

```diff
@@ -23,66 +23,66 @@ def test_new_deposit_under_min_activation_balance(spec, state):
     amount = spec.MIN_ACTIVATION_BALANCE - 1
     deposit = prepare_state_and_deposit(spec, state, validator_index, amount, signed=True)
 
-    yield from run_deposit_processing_eip7251(spec, state, deposit, validator_index)
+    yield from run_deposit_processing_electra(spec, state, deposit, validator_index)
 
 
-@with_eip7251_and_later
+@with_electra_and_later
 @spec_state_test
 def test_new_deposit_min(spec, state):
     # fresh deposit = next validator index = validator appended to registry
     validator_index = len(state.validators)
     amount = spec.MIN_DEPOSIT_AMOUNT
     deposit = prepare_state_and_deposit(spec, state, validator_index, amount, signed=True)
-    yield from run_deposit_processing_eip7251(spec, state, deposit, validator_index)
+    yield from run_deposit_processing_electra(spec, state, deposit, validator_index)
 
 
-@with_eip7251_and_later
+@with_electra_and_later
 @spec_state_test
 def test_new_deposit_between_min_and_max(spec, state):
     # fresh deposit = next validator index = validator appended to registry
     validator_index = len(state.validators)
-    amount = spec.MAX_EFFECTIVE_BALANCE_EIP7251 // 2
+    amount = spec.MAX_EFFECTIVE_BALANCE_electra // 2
```

Comment:
Looks like a find/replace mistake here.

Reply:
ah, I haven't updated the tests yet! kind blocked on #3656